packages/reflect-core/lib/radial-gradient/radial-gradient.manifest.ts

```diff
@@ -0,0 +1,49 @@
+import type { Color } from "../color";
+import { Alignment } from "../alignment";
+import { GradientType } from "../gradient";
```

Comment:
use `import *type* { GradientType } from "../gradient";`

Reply:
think it would work since it enum (guessing)

Reply:
Since enum is not a `type` like the ```import type { Color } from "../color";``` used above, it should not be imported as a type.